### PR TITLE
feat(welcomer): setup multiple responses

### PR DESCRIPTION
**Pull request by Moore2021**
Be able to set multiple welcome messages based on what channels a user can see

### CODE REVIEW

File: src/events/guildMemberUpdate.js [hunk 1]
```diff
@@ -1,29 +1,101 @@
...
+                    simplified: true,
+                    prebuffer: true,
+                    image: renderedBanner
+                })
+                .catch(() => client.logger.warn(`${instance} failed to send the requested welcomer message due to there was no welcomer channel and the user's dm were locked.`))
+        } else {
+            /**
+             * Handle multiple channels
+             */
+            if (configs.get(`WELCOMER_ADDITIONAL_CHANNELS`)) {
```
Comment (klerikdust) [inline src/events/guildMemberUpdate.js:64]: ` if (configs.get('WELCOMER_ADDITIONAL_CHANNELS')) {`
you missed the `.value` property here; otherwise, this condition will always stay true lol

File: src/events/guildMemberUpdate.js [hunk 2]
```diff
@@ -1,29 +1,101 @@
...
+                let additional_channels = configs.get(`WELCOMER_ADDITIONAL_CHANNELS`).value
```
Comment (klerikdust) [inline src/events/guildMemberUpdate.js:65]: May be an unnecessary change, but could you update the variable name to use `camelCase` format? to follow with the rest of the variables

File: src/events/guildMemberUpdate.js [hunk 3]
```diff
@@ -1,29 +1,101 @@
...
+                        if (!guild.channels.cache.has(channel)) return client.logger.warn(`${instance} failed to send welcomer message due to invalid target channel in GUILD_ID:${guild.id}`)
+                        getWelcomerText = parseWelcomerText(text, guild, newMember)
+                        const ch = guild.channels.cache.get(channel)
+
+                        client.responseLibs(ch, true)
+                            .send(getWelcomerText, {
+                                simplified: true,
+                                prebuffer: true,
+                                image: renderedBanner
+                            })
```
Comment (klerikdust) [inline src/events/guildMemberUpdate.js:81]: Add additional `.catch()` error log here, just to make it less headache later on

**Review by klerikdust — CHANGES_REQUESTED** [review]
Require a few minor changes @Moore2021

**Comment by Moore2021** [comment]
just pushed changes
